qt/pubnub.hpp

```diff
@@ -279,21 +280,48 @@ namespace pubnub {
         /// Returns the current `auth` key for this context
         std::string auth() const { return d_pbqt.auth().toStdString(); }
 
-        /** Sets the UUID to @p uuid. If @p uuid is an empty string,
-            UUID will not be used.
-         */
-        void set_uuid(std::string const &uuid) {
-            d_pbqt.set_uuid(QString::fromStdString(uuid));
-        }
-        /// Set the UUID to a random-generated one
-        int set_uuid_v4_random() {
-            d_pbqt.set_uuid_v4_random();
-            return 0;
-        }
-        /// Returns the current UUID
-        std::string uuid() const { return d_pbqt.uuid().toStdString(); }
-
-        /// Returns the next message from the context. If there are
+        /** Sets the user_id to @p uuid. If @p uuid is an empty string,
+            user_id will not be used.
+        
+	    @deprecated this is provided as a workaround for existing users.
+	    Please use `set_user_id` instead.
+  	*/
+        PUBNUB_DEPRECATED void set_uuid(std::string const &uuid) {
+            set_user_id(uuid);
+	}
+
+	/** Sets the user_id to @p user_id. if @p user_id is an empty string,
+            user_id will not be used.
+        */
+    	void set_user_id(std::string const& user_id)
+    	{
+            d_pbqt.set_user_id(QString::fromStdString(uuid));
+    	}
+
+	/// Set the user_id with a random-generated UUID
+        /// 
+	/// @deprecated this is provided as a workaround for existing users.
+        /// Please use `set_user_id_with_random_uuid_v4` instead.
+   	PUBNUB_DEPRECATED int set_uuid_v4_random() {
```

Comment:
Can you explain to me why sometimes we have `/** */` and sometimes `///`? It's nothing major obviously, I'm just curious

Reply:
No idea. 

I'm following the rules that most fit into current file. 

I think that I will merge comment styles in all repository but this PR was made at the beginning of my contributions se I wasn't brave enough to do such a change.